src/stdlib/typedarray.md

```diff
@@ -138,6 +138,11 @@ The TypedArray API works very much like JavaScript's \([MDN](https://developer.m
   ```
   Reverses an array's values in place, modifying the array before returning it.
 
+* ```ts
+  function set(source: TypedArray, fromIndex?: i32): void
+  ```
+  Sets the array values (started at `fromIndex`, or 0) based on the contents of another array.
+
```

Comment:
```suggestion
  function set(source: TypedArray, offset?: i32): void
  ```
  Sets the typed array values (started at `offset`, or 0) reading input values from a specified `source` array or typed array.

```